### PR TITLE
Issue 1327 additional ekm updates

### LINKED ISSUE #1327: additional EKM updates to the app
part of #1298 after #1168

when the OrgRules show that EKM is in use:
 - [x] forbid adding keys in settings -> keys
 - [x] forbid updating or modifying keys in settings -> keys
 - [x] forbid removing keys in settings -> keys
 - [x] forbid exporting or showing private keys

### PR DESCRIPTION
**Pull request by DenBond7**
This PR added additional ekm updates

close #1327 // if this PR closes an issue

----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities

### CODE REVIEW

**Comment by IvanPizhenko** [comment]
@DenBond7 why do you add label "PR Submitted" to pull requests? I have introduced this label with intent to put it on issues to help myself to see what issues in the list of issues assigned to me are already done (but not merged & closed yet), and what are not and so can be potentially be the next thing to work on.

File: FlowCrypt/src/androidTest/java/com/flowcrypt/email/util/AccountDaoManager.kt [hunk 1]
```diff
@@ -40,6 +41,11 @@ class AccountDaoManager {
       return getUserFromBaseSettings("no_backups@flowcrypt.test")
     }
 
+    fun getUserWithOrgRules(orgRules: OrgRules): AccountEntity {
```
Comment (IvanPizhenko) [inline FlowCrypt/src/androidTest/java/com/flowcrypt/email/util/AccountDaoManager.kt:44]: I suggest to rename this into `getTestUserWithOrgRules`.

**Review by IvanPizhenko — CHANGES_REQUESTED** [review]

**Comment by DenBond7** [comment]
> @DenBond7 why do you add label "PR Submitted" to pull requests? I have introduced this label with intent to put it on issues to help myself to see what issues in the list of issues assigned to me are already done (but not merged & closed yet), and what are not and so can be potentially be the next thing to work on.

Ah... Sorry, I didn't know. I thought you have created it to mark pull requests. No prob. From now, I will not use it.

File: FlowCrypt/src/androidTest/java/com/flowcrypt/email/util/AccountDaoManager.kt (hunk 1 above)
Comment (DenBond7) [reply FlowCrypt/src/androidTest/java/com/flowcrypt/email/util/AccountDaoManager.kt:44]: I'm not sure. This class is located in the tests package. So all things there relate to testing. And adding `Test` just will increase the method name.

If you don't have any new questions or recommendations please approve the current PR. It is blocking me a little to complete another one.

Comment (IvanPizhenko) [reply FlowCrypt/src/androidTest/java/com/flowcrypt/email/util/AccountDaoManager.kt:44]: ok, have not noticed it is in the test package

**Review by IvanPizhenko — APPROVED** [review]

**Pull request merged**